### PR TITLE
Adding Save As node

### PR DESCRIPTION
**Pull request by kraftwerk15**
Added a save as node that takes in a document and file path.

### CODE REVIEW

File: src/Rhythm/Revit/Application/Application.cs [hunk 1]
```diff
@@ -106,5 +106,25 @@ public static List<Autodesk.Revit.DB.Document> GetOpenDocuments(bool runIt)
             }
             return documents;
         }
+
+        /// <summary>
+        /// This node will save the Revit document to another path.
+        /// </summary>
+        /// <param name="Document">A valid Revit Document.</param>
```
Comment (johnpierson) [inline src/Rhythm/Revit/Application/Application.cs:113]: lowercase "d" for input names. The standard is camelCase.

File: src/Rhythm/Revit/Application/Application.cs [hunk 2]
```diff
@@ -106,5 +106,25 @@ public static List<Autodesk.Revit.DB.Document> GetOpenDocuments(bool runIt)
...
+        /// <param name="FilePath">The file path to save the document.</param>
```
Comment (johnpierson) [inline src/Rhythm/Revit/Application/Application.cs:114]: filePath

File: src/Rhythm/Revit/Application/Application.cs [hunk 3]
```diff
@@ -106,5 +106,25 @@ public static List<Autodesk.Revit.DB.Document> GetOpenDocuments(bool runIt)
...
+        /// <returns name="result">A string message whether the save as was successful or a failure.</returns>
+        [NodeCategory("Action")]
+        public static string SaveAsDocument(Revit.Application.Document Document, string FilePath)
```
Comment (johnpierson) [inline src/Rhythm/Revit/Application/Application.cs:117]: rename the method to just SaveAs. This would allow it to show in Dynamo as `Documents.SaveAs`

Comment (johnpierson) [inline src/Rhythm/Revit/Application/Application.cs:117]: document, filePath

**Comment by kraftwerk15** [comment]
Addressed changes in latest update.

**Pull request merged**